antenna/configlib.py

```diff
@@ -0,0 +1,323 @@
+"""This module contains the configuration infrastructure allowing for deriving
+configuration from a .ini file and the process environment.
+
+In order of precedence:
+
+1. settings overrides (usually only applies when running tests)
+2. process environment
+3. .ini file (specified by ANTENNA_INI environment variable)
+
+Example of usage::
+
+    from antenna.configlib import config
+
+    DEBUG = config('DEBUG', default=True, parser='bool')
```

Comment:
With the custom parser, you can do that right now:

```
DATABASES = config('DATABASES', default='postgres://foo:bar@baz.com:5432', parser=dj_database_url)
```

My point is that we can't guess what the parser should be based on the type of the default. In some cases it works, but not all. Given that, seems better to keep it consistent and make it always required.